scootapi/client/run_job_cmd.go

```diff
@@ -22,8 +25,9 @@ func (c *runJobCmd) registerFlags() *cobra.Command {
 		Use:   "run_job",
 		Short: "run a job",
 	}
-	r.Flags().StringVar(&c.snapshotId, "snapshot_id", scoot.TaskDefinition_SnapshotId_DEFAULT, "snapshot ID to run job against")
-	r.Flags().StringVar(&c.jobFilePath, "job_def", "", "JSON file to read jobs from")
+	r.Flags().StringVar(&c.snapshotId, "stream_name", "sm", "If passing snapshot_id=SHA, this is the global UID for the associated repo.")
```

Comment:
I added this for dx-team ease-of-use, so it's more likely a candidate for internal docs. When we move beyond CI+sickle as our only customer we'll want to do more user facing docs.